gapis/api/templates/cpp_common.tmpl

```diff
@@ -178,6 +178,7 @@
 {{define "C++.CallParameters"}}
   {{AssertType $ "Function"}}
 
+  {{Global "CallFunction" $}}
```

Comment:
This will break if you have a call in a call. Use hacky stack to fix this:

```
  {{$prevCallFunction := Global "CallFunction"}}
  {{Global "CallFunction" $}}
  {{ForEach $.CallParameters "C++.Parameter" | JoinWith ", "}}
  {{Global "CallFunction" $prevCallFunction}}
```

Reply:
added in 0535813